tests/rest/admin/test_user.py

```diff
@@ -1768,3 +1768,111 @@ def test_mau_limit(self):
         # though the MAU limit would stop the user doing so.
         puppet_token = self._get_token()
         self.helper.join(room_id, user=self.other_user, tok=puppet_token)
+
+
+class WhoisRestTestCase(unittest.HomeserverTestCase):
```

Comment:
Thanks for adding tests here! I wonder if we need to each test to both URLs, but I don't see any harm in including them.